charts/osm-arc/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Default values for osm-arc.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+OpenServiceMesh:
+    ignoreNamespaces: "kube-system azure-arc arc-osm-system"
+
+osm:
+  OpenServiceMesh:
+    enableDebugServer: true
+    enablePermissiveTrafficPolicy: true
+    enableFluentbit: true
+    image:
+      registry: mcr.microsoft.com/oss/openservicemesh
+    fluentBitImage:
+      name: fluentbit-logger
+      registry: "mcr.microsoft.com/oss/fluent"
+      tag: "v1.6.2"
+      pullPolicy: IfNotPresent
+    enforceSingleMesh: true
+    enablePrometheus: false
+    deployJaeger: false
+    customLogAnalyticsWorkspace: false
+    customerID: 49a4d43a-81ef-4607-83f6-a13872c6ae60
```

Comment:
You should include these new fields in this values.yaml _after_ the osm dependency chart has been upgraded to 0.5.1 as they dont exist in 0.5.0 right?